src/components/ActionDataSamplesCard.tsx

```diff
@@ -154,28 +162,78 @@ const RecordingArea = ({
 }: {
   action: GestureData;
   selected: boolean;
-  onRecord: () => void;
+  onRecord: (recordingsToCapture: number, continuousRecording: boolean) => void;
 }) => {
   const intl = useIntl();
   return (
     <VStack w="8.25rem" justifyContent="center">
-      <Button
-        variant={selected ? "solid" : "outline"}
-        colorScheme="red"
-        onClick={onRecord}
-        aria-label={intl.formatMessage(
-          { id: "record-action-aria" },
-          { action: action.name }
-        )}
-      >
-        <FormattedMessage id="record-action" />
-      </Button>
+      <Menu>
+        <ButtonGroup isAttached>
+          <Button
+            pr={2}
+            variant={selected ? "solid" : "outline"}
+            colorScheme="red"
+            onClick={() => onRecord(1, false)}
+            aria-label={intl.formatMessage(
+              { id: "record-action-aria" },
+              { action: action.name }
+            )}
+          >
+            <FormattedMessage id="record-action" />
+          </Button>
+          <MoreMenuButton
+            minW={8}
+            variant={selected ? "record" : "recordOutline"}
+            aria-label="More record options"
```

Comment:
TODO: Text needs to be translatable.

Reply:
Maybe "More recording options" ?

Reply:
[4e902a9](https://github.com/microbit-foundation/ml-trainer/pull/450/commits/4e902a9793695c4bfd5b3c72bce1f1cf08d3dee8)